### PR TITLE
opengl: use libglvnd on freebsd

### PR DESCRIPTION
**Pull request by ericLemanissier**
https://cgit.freebsd.org/ports/commit/?id=f0559e873477fc1b4d625ca9edf8c4a0f3cc2489

Specify library name and version:  **opengl/system**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by ericLemanissier** [comment]
test passed successfully on https://github.com/bincrafters/system-packages-checks/runs/3046132642?check_suite_focus=true#step:3:12687

File: recipes/opengl/all/conanfile.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def system_requirements(self):
             else:
                 self.output.warn("Don't know how to install OpenGL for your distro.")
         elif tools.os_info.is_freebsd and self.settings.os == "FreeBSD":
-            packages = ["mesa-libs"]
+            packages = ["libglvnd"]
```
Comment (Croydon) [inline recipes/opengl/all/conanfile.py:61]: Is this available in all versions of the last few years?

**Review by Croydon — APPROVED** [review]

File: recipes/opengl/all/conanfile.py (hunk 1 above)
Comment (ericLemanissier) [reply recipes/opengl/all/conanfile.py:61]: my source is https://pkgs.org/search/?q=libglvnd, it shows both FreeBSD 12 and 13 have it, but they are the only versions available on this website :/

Comment (Croydon) [reply recipes/opengl/all/conanfile.py:61]: Seems like FreeBSD 12 was published in December 2018, but the package was added 2020-05-27 according to https://cgit.freebsd.org/ports/log/graphics/libglvnd

Maybe they backport packages to still supported versions? Anyway, we can still improve this later if there is demand for it I guess

**Review by uilianries — APPROVED** [review]

**Review by prince-chrismc — APPROVED** [review]

**Pull request merged**